lib/ts/Scrubber.h

```diff
@@ -0,0 +1,89 @@
+/** @file
+
+  The Scrubber class helps replace patterns of text inside a buffer with
+  replacement text.
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+#pragma once
+#include <vector>
+#include "pcre.h"
+
+#include "ts/ink_assert.h"
+#include "ts/ink_memory.h"
+#include "ts/MemView.h"
+
+struct Scrub {
+  ~Scrub()
+  {
+    if (compiled_re) {
+      pcre_free(const_cast<pcre *>(compiled_re));
+    }
+  }
+  static const int OVECCOUNT = 30;
```

Comment:
It was used in the PCRE example documentation. Really this could be any value >= 3.